src/_guides/libraries/create-library-packages.md

```diff
@@ -121,8 +121,7 @@ This adapter handles HttpRequest objects from `dart:io`.
 
 ## Importing library files
 
-When importing a library file from another package, use the
-the `package:` directive to specify the URI of that file.
+When importing a library file from another package, use the `package:` directive to specify the URI of that file.
```

Comment:
We aim to keep each line <= 80 characters and to use some sort of semantic line break: https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks

```suggestion
When importing a library file from another package, use
the `package:` directive to specify the URI of that file.
```